README.md

```diff
@@ -60,17 +33,24 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 bookstore-react-redux<a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
 
-**[your_project__name]** is a...
+**bookstore-react-redux** is a project I created using ReactJS. From this project anticipate to learn:
+- React 18 and best practices
+- props and states
+- React Router 
+- React Icons 
+- Re-Usable Components
+- React Hooks (useState)
+- CSS transition and animation
+- Redux for state management
+- and more...
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
 
 <details>
   <summary>Client</summary>
```

Comment:
- Please, kindly remove the server and database, as they're not used in this project to make your readme more consistent with the project you've built.